docs/quickstart/first-setup.md

```diff
@@ -0,0 +1,27 @@
+<h1>How to Setup a Openshock Shocker</h1>
+<h2>Setup ESP</h2>
+<br/>
+<br/>
+1. To Setup your ESP (this is the Communication device between the API and your Shocker) you have to connect it to a power source, like a USB powerbrick or your PC USB Port (ESP Board has USB-C).<br/>
+2. After you plugged it in grab your phone and search for a WiFi network called similar to "Openshock-24:DC:C3:9F:72:C8" and connect to it.<br/>
+3. Now open your Phones browser and type in 10.10.10.10 or openshock.local this should give you a webinterface for the ESP.<br/>
```

Comment:
You can add stuff around things to make them stand out in multiple ways.

Here I would use \`\`this\`\` to make it look like ``this``

Example:
> Markdown go brr, look funny strings ``10.10.10.10`` or ``openshock.local``